screening/openg2p.json

```diff
@@ -0,0 +1,97 @@
+{
+  "name": "OpenG2P",
+  "ownership": {
+    "isOwnershipExplicit": "Yes",
+    "copyrightURL": "https://docs.openg2p.org/"
+  },
+  "dependencies": {
+    "mandatoryDepsCreateMoreRestrictions": "No",
+    "isSoftwarePltIndependent": "",
+    "pltIndependenceDesc": ""
+  },
+  "documentation": {
+    "isDocumentationAvailable": "Yes",
+    "documentationURL": [
+      "https://docs.openg2p.org/"
+    ]
+  },
+  "NonPII": {
+    "collectsNonPII": "Yes",
+    "nonPIIAccessMechanism": "Yes"
+  },
+  "privacy": {
+    "isPrivacyCompliant": "Yes",
+    "privacyComplianceList": [
+      "OpenG2P is a tool used by adopters, the OpenG2P foundation by itself does not store any PII"
+    ],
+    "isLocIntlAdherent": "Yes",
+    "locIntlComplianceList": [
+      "OpenG2P is a tool used by adopters, the OpenG2P foundation by itself does not store any PII"
```

Comment:
This answer is not valid here. If they claim to be compliant with local and international laws, then they need to provide supporting evidence.